Accessibility: add Overview topic (#5042)
DevExpress/devextreme-documentation: concepts/Common/Accessibility/00 Overview/10 Keyboard Navigation Support.md

> DevExtreme components support keyboard navigation, which allows users who rely on alternative input methods to navigate and interact with the content effectively.

Reviewer: ```suggestion
DevExtreme components support keyboard navigation that allows users who rely on alternative input methods to navigate and interact with the content.
```

> You can try to use keyboard to navigate in the following demos:

Reviewer: ```suggestion
You can use the keyboard to navigate in the following demos:
```
